Allow Sphinx to recognize the Sage prompt "sage:"
the same way it recognizes the ">>>" from Python

diff -ru a/sphinx/highlighting.py b/sphinx/highlighting.py
--- a/sphinx/highlighting.py	2016-03-28 15:16:34.000000000 +0200
+++ b/sphinx/highlighting.py	2016-04-22 22:34:29.642929758 +0200
@@ -96,15 +96,19 @@
             source = source.decode()
 
         # find out which lexer to use
-        if lang in ('py', 'python'):
+        elif lang in ('py', 'python'):
             if source.startswith('>>>'):
                 # interactive session
                 lexer = lexers['pycon']
+            elif source.startswith('sage: '):
+                lexer = lexers['pycon']
             else:
                 lexer = lexers['python']
         elif lang in ('py3', 'python3', 'default'):
             if source.startswith('>>>'):
                 lexer = lexers['pycon3']
+            elif source.startswith('sage: '):
+                lexer = lexers['pycon']
             else:
                 lexer = lexers['python3']
         elif lang == 'guess':
